### PR TITLE
feat(onboarding-ui): CloudOrganizationInfo Page

### PR DESCRIPTION
**Pull request by dougfabris**
- [x] I have read the Contributing Guide
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have labeled the PR correctly with the related package
- [x] I have run Loki's visual regression tests (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by tassoevan — CHANGES_REQUESTED** [review]
Something is not right with the "Next" button when there is option to skip the wizard step.

File: packages/onboarding-ui/src/forms/OrganizationInfoForm/OrganizationInfoForm.tsx [hunk 1]
```diff
@@ -165,9 +170,20 @@ const OrganizationInfoForm = ({
           <Button onClick={onBackButtonClick}>
             {t('component.form.action.back')}
           </Button>
+
           <Button type='submit' primary disabled={isValidating || isSubmitting}>
-            {t('component.form.action.next')}
+            {onConfirmText || t('component.form.action.next')}
```
Comment (tassoevan) [inline packages/onboarding-ui/src/forms/OrganizationInfoForm/OrganizationInfoForm.tsx:175]: Why not `??`?

```suggestion
            {onConfirmText ?? t('component.form.action.next')}
```

It's odd to imagine a valid falsy value here, but...

File: packages/onboarding-ui/src/forms/OrganizationInfoForm/OrganizationInfoForm.tsx [hunk 2]
```diff
@@ -28,9 +29,11 @@ type OrganizationInfoFormProps = {
   organizationIndustryOptions: SelectOptions;
   organizationSizeOptions: SelectOptions;
   countryOptions: SelectOptions;
+  onConfirmText?: string;
```
Comment (tassoevan) [inline packages/onboarding-ui/src/forms/OrganizationInfoForm/OrganizationInfoForm.tsx:32]: For the sake of flexibility:

```suggestion
  onConfirmText?: ReactNode;
```

Comment (dougfabris) [reply packages/onboarding-ui/src/forms/OrganizationInfoForm/OrganizationInfoForm.tsx:32]: I'll remember that!

File: packages/onboarding-ui/src/pages/OrganizationInfoPage/OrganizationInfoPage.tsx [hunk 3]
```diff
@@ -16,13 +18,15 @@ type OrganizationInfoPageProps = {
   initialValues?: OrganizationInfoPayload;
   onSubmit: SubmitHandler<OrganizationInfoPayload>;
   onBackButtonClick: () => void;
+  onConfirmText?: string;
```
Comment (tassoevan) [inline packages/onboarding-ui/src/pages/OrganizationInfoPage/OrganizationInfoPage.tsx:21]: Better keep `on` prefixes for event callback props?

**Review by tassoevan — CHANGES_REQUESTED** [review]

File: packages/onboarding-ui/src/pages/OrganizationInfoPage/OrganizationInfoPage.tsx (hunk 3 above)
Comment (dougfabris) [reply packages/onboarding-ui/src/pages/OrganizationInfoPage/OrganizationInfoPage.tsx:21]: Sounds better, thnks!

**Comment by dougfabris** [comment]
> Something is not right with the "Next" button when there is option to skip the wizard step.

Solved!

**Review by tassoevan — APPROVED** [review]

**Pull request merged**